roxie/udplib/udptrr.cpp

```diff
@@ -679,6 +712,8 @@ class CReceiveManager : implements IReceiveManager, public CInterface
 
     int free_slots()
     {
+        // mck - skip for now, several attempts at keeping track of inflight do not seem accurate enough and cause
+        //       too few packets to be sent as time goes on.  TODO - look into decay of inflight by X pkts/s ...
```

Comment:
The inflight code may not work, but does removing it just reintroduce the previous problems with lost packets?  Maybe that's ok as long as there is a plan to revisit it soon.

Reply:
Ideally, getting Aeron working is the plan...